File: spring-grpc-client/app/src/main/java/com/example/springgrpc/app/SpringGrpcClientApplication.java
package com.example.springgrpc.app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class SpringGrpcClientApplication {
    public static void main(String[] args) {
        SpringApplication.run(SpringGrpcClientApplication.class, args);
    }
}

File: spring-grpc-client-kotlin/app/src/main/kotlin/com/example/springgrpc/app/SpringGrpcClientApplication.kt
package com.example.springgrpc.app

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class SpringGrpcClientApplication

fun main(args: Array<String>) {
    runApplication<SpringGrpcClientApplication>(*args)
}
